src/main/java/rx/internal/schedulers/NewThreadWorker.java

```diff
@@ -30,24 +34,111 @@
     private final ScheduledExecutorService executor;
     private final RxJavaSchedulersHook schedulersHook;
     volatile boolean isUnsubscribed;
-
+    /** The purge frequency in milliseconds. */
+    private static final String FREQUENCY_KEY = "io.reactivex.rxjava.scheduler.jdk6.purge-frequency-millis";
```

Comment:
Hystrix could use the enhancements in #2579 and since it affects the same file, it is much easier to change the values there in a plain commit than rebasing.